backend/de.metas.manufacturing/src/main/sql/postgresql/system/30-de.metas.adempiere.libero/5727710_sys_gh18356_Uncomplete_PP_Order.sql

```diff
@@ -0,0 +1,22 @@
+-- Value: Produktionsauftrag zurücksetzen
+-- Classname: de.metas.process.ExecuteUpdateSQL
+-- 2024-07-02T14:52:16.758Z
+INSERT INTO AD_Process (AccessLevel,AD_Client_ID,AD_Org_ID,AD_Process_ID,AllowProcessReRun,Classname,CopyFromProcess,Created,CreatedBy,Description,EntityType,IsActive,IsApplySecuritySettings,IsBetaFunctionality,IsDirectPrint,IsFormatExcelFile,IsNotifyUserAfterExecution,IsOneInstanceOnly,IsReport,IsTranslateExcelHeaders,IsUpdateExportDate,IsUseBPartnerLanguage,LockWaitTimeout,Name,PostgrestResponseFormat,RefreshAllAfterExecution,ShowHelp,SpreadsheetFormat,SQLStatement,Type,Updated,UpdatedBy,Value) VALUES ('3',0,0,585403,'Y','de.metas.process.ExecuteUpdateSQL','N',TO_TIMESTAMP('2024-07-02 17:52:15','YYYY-MM-DD HH24:MI:SS'),100,'Update PP_Order.PlanningStatus=''P''','D','Y','N','N','N','Y','N','N','N','Y','N','Y',0,'Produktionsauftrag zurücksetzen','json','N','N','xls','UPDATE pp_order SET planningstatus = ''P'' WHERE pp_order_id = @PP_Order_ID/NULL@;','SQL',TO_TIMESTAMP('2024-07-02 17:52:15','YYYY-MM-DD HH24:MI:SS'),100,'Produktionsauftrag zurücksetzen')
```

Comment:
what's the reason for implementing this as an SQL process instead of Java process which calls some API that un-closes the manufacturing order?

Reply:
saw that in the initial concept.... have to clarify with the RE/PO